README.md

```diff
@@ -53,10 +53,9 @@ Here you can find the documentation for different versions:
 - Latest stable version: [GitHub docs](https://github.com/flame-engine/flame/tree/master-v0.x/doc)
 - Latest v1.0.0 version: [GitHub docs](https://github.com/flame-engine/flame/tree/1.0.0-rc8/doc)
 
-The complete documentation can be found [here](doc/README.md).
+The complete documentation can be found [here](README.md).
 
-Many examples of different features can be found [here](doc/examples) and a good starting example
-can be found [here](/example).
+We provide many online examples of different features which can be played from your browser [here](https://flame-engine.github.io/flame/).
```

Comment:
Max 100 in line length

Reply:
I think "online" can be removed, and "played" maybe replaced by "tried out"?

Reply:
Both fixed